code/addons/storysource/package.json

```diff
@@ -66,6 +89,14 @@
   "publishConfig": {
     "access": "public"
   },
+  "bundler": {
+    "entries": [
+      "./src/events.ts",
+      "./src/index.ts",
+      "./src/manager.tsx",
+      "./src/StoryPanel.tsx"
```

Comment:
I don't think `StoryPanel` needs to be its own entry and exports, since nothing else outside of this project imports it, it's only used internally within the package.

Same goes for `events.ts`, that's re-exported by `index.ts` already.

Reply:
I didn't create the `manager.js` on the root before because I saw that `register.js` was doing the same function `import './dist/manager'`, but I guess that was the error. I created the new file and is working locally now. Thank you for the help. I am going to see if the new changes fixes the CI checks.